app/packages/scrubber/scrubber.coffee

```diff
@@ -117,6 +122,8 @@ Template.scrubber.events
     event.stopImmediatePropagation()
     event.stopPropagation()
   'click .scrubber-play': (event, instance) ->
+    if ($(event.target).hasClass('.scrubber-play-disabled'))
+      return
```

Comment:
Looks like we need to use `$(event.currentTarget)` since sometimes the user clicks on the `<i>` within `.scrubber-play` which won't have the class.
And I don't think you need the `.` when calling `hasClass`.